src/csharp/Microsoft.Spark/Services/ConfigurationService.cs

```diff
@@ -62,19 +65,27 @@ public string GetWorkerExePath()
                 return _workerPath;
             }
 
+            var procName = s_procFileName;
```

Comment:
`string procName =`

Reply:
Can we also change
```C#
private static readonly string s_procBaseFileName = "Microsoft.Spark.Worker";
private static readonly string s_procFileName =
```
to
```C#

private static readonly string s_workerBaseName = "Microsoft.Spark.Worker";
private static readonly string s_workerName =
```

Reply:
Once that is done, you can change this code block to:
```C#
string workerName = Environment.GetEnvironmentVariable(WorkerNameEnvVarName);
if (string.IsNullOrEmpty(workerName))
{
    workerName = s_workerName;
}
```

and use `workerName` in place of `procName` below.